### PR TITLE
OEL-2606: Add a link back to the glossary page in breadcrumb

### PR DESCRIPTION
**Pull request by AaronGilMartinez**
### Description

[Insert description here]

### Change log

- Added:
- Changed:
- Deprecated:
- Removed:
- Fixed:
- Security:

### Commands

```sh
[Insert commands here]

```

### CODE REVIEW

File: modules/oe_showcase_glossary/config/install/pathauto.pattern.glossary_term_url_alias_pattern.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+langcode: en
```
Comment (drishu) [inline modules/oe_showcase_glossary/config/install/pathauto.pattern.glossary_term_url_alias_pattern.yml:1]: add this to the config_devel in info.yml

File: modules/oe_showcase_glossary/tests/src/ExistingSite/GlossaryTest.php [hunk 2]
```diff
@@ -310,8 +390,11 @@ protected function generateTerms(): array {
     $initials['ç'] = rand(1, 5);
     // Add a special character.
     $initials['Æ'] = rand(1, 5);
-    // Add one entry with more than 20 terms in it, to trigger paging.
-    $initials[array_rand($initials)] = 41;
+    // And Add one entry with more than 20 terms in it, to trigger paging.
+    $initials[array_rand(array_diff($initials, ['a']))] = 41;
+    // Test pager in 'a' character, to check aditional URL parameters like
```
Comment (drishu) [inline modules/oe_showcase_glossary/tests/src/ExistingSite/GlossaryTest.php]:
```suggestion
    // Test pager in 'a' character, to check additional URL parameters like
```

**Review by drishu — CHANGES_REQUESTED** [review]

File: modules/oe_showcase_glossary/config/install/pathauto.pattern.glossary_term_url_alias_pattern.yml (hunk 1 above)
Comment (AaronGilMartinez) [reply modules/oe_showcase_glossary/config/install/pathauto.pattern.glossary_term_url_alias_pattern.yml:1]: Added

File: modules/oe_showcase_glossary/tests/src/ExistingSite/GlossaryTest.php (hunk 2 above)
Comment (AaronGilMartinez) [reply modules/oe_showcase_glossary/tests/src/ExistingSite/GlossaryTest.php]: Fixed

**Review by drishu — DISMISSED** [review]

File: modules/oe_showcase_glossary/tests/src/ExistingSite/GlossaryTest.php [hunk 3]
```diff
@@ -377,4 +460,33 @@ protected function sortTerms(array &$terms): void {
...
   }
 
+  /**
+   * Asserts breadcrumbs in the current page.
+   *
+   * @param array $expected_segments
+   *   Associative array with text and URL for the breadcrumbs segments.
+   */
+  protected function assertBreadcrumbs(array $expected_segments): void {
+    $crawler = new Crawler($this->getSession()->getPage()->getContent());
```
Comment (brummbar) [inline modules/oe_showcase_glossary/tests/src/ExistingSite/GlossaryTest.php]: Don't use the crawler here. Use the assert session.

**Review by brummbar — CHANGES_REQUESTED** [review]

File: modules/oe_showcase_glossary/tests/src/ExistingSite/GlossaryTest.php (hunk 3 above)
Comment (AaronGilMartinez) [reply modules/oe_showcase_glossary/tests/src/ExistingSite/GlossaryTest.php]: Replaced

**Review by drishu — DISMISSED** [review]

File: modules/oe_showcase_glossary/tests/src/ExistingSite/GlossaryTest.php [hunk 4]
```diff
@@ -377,4 +459,32 @@ protected function sortTerms(array &$terms): void {
...
+   */
+  protected function assertBreadcrumbs(array $expected_segments): void {
+    $assert_session = $this->assertSession();
+
+    // Check that the breadcrumbs block is present in the page.
+    $breadcrumbs_block = $assert_session->elementExists('css', '#block-oe-showcase-theme-breadcrumbs');
+
+    // Check segments number.
+    $breadcrumbs_items = $breadcrumbs_block->findAll('css', 'li.breadcrumb-item');
+    $this->assertCount(count($expected_segments), $breadcrumbs_items);
```
Comment (brummbar) [inline modules/oe_showcase_glossary/tests/src/ExistingSite/GlossaryTest.php]: assertSameSize()

**Review by brummbar — CHANGES_REQUESTED** [review]

**Review by brummbar — APPROVED** [review]

**Review by drishu — APPROVED** [review]

**Pull request merged**